clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/ops/DeployAppengineAtomicOperation.groovy

```diff
@@ -117,9 +122,37 @@ class DeployAppengineAtomicOperation implements AtomicOperation<DeploymentResult
 
     task.updateStatus BASE_PHASE, "Deploying version $versionName..."
     jobExecutor.runCommand(deployCommand)
+    deleteFiles(writtenFullConfigFilePaths)
```

Comment:
Any chance `jobExecutor` will throw an exception? If so, `deleteFiles` should probably live in a `finally` block so we don't leave old configs dangling.

Also, doesn't `runCommand` run asynchronously? If so, it's possible the files will be deleted before the command is able to read them.